.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,76 @@
+name: Bug Report
+description: Report a bug impacting ZAP.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report a ZAP bug! Fill out the form below to provide us with info to help fix the bug. 
+        Only use this if you're 100% sure you've found a bug and can reproduce it. If you're looking for general help with ZAP, try the [User Group](https://groups.google.com/g/zaproxy-users).
```

Comment:
Add something like: "Please do not ask questions as issues - these will be closed." ?

Reply:
Sure